template/.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+on: [push]
+
+jobs:
+  static-analysis-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12.2'
+        cache: 'poetry'
+
+    - name: Install dependencies
+      run: |
+        pip install invoke
+        pip install poetry
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** You could install both these package with a single call to pip:

```shell
pip install invoke poetry
```

Reply:
Updated in https://github.com/ackama/django-template/commit/c31cf55160c355420fa8a874e1795922c1d1ccd6